.circleci/config.yml

```diff
@@ -615,6 +615,14 @@ jobs:
           pattern: "grpc"
           snapshot: true
 
+  grpc_aio:
+    <<: *machine_executor
+    parallelism: 6
+    steps:
+      - run_test:
+          pattern: "grpc_aio"
+          snapshot: true
```

Comment:
you probably don't need this, since the `grpc_aio` tests will get picked up by `pattern: "grpc"` above.

or we can do:

```
pattern: "^grpc$"

pattern: "^grpc_aio$"
```

to separate them

Reply:
🤔 that didn't seem to even fix it. one of us will have to try this locally and see if we can figure out why it isn't finding/running any of the `grpc_aio` test cases.

Reply:
Now I got why.

`scripts/tested_pythons` says there are 7 python versions (i.e. 2.7, 3.5-3.10) and `circleci tests split` splits them between workers so when parallelism is 6, 1 worker processes almost 1 python version. Since not all of the test results are shown in the CIrcleCI UI and `grpc_aio` job does not have tests for 2.7 and 3.5, CI ends up 0 executed tests.

After I reduced the parallelism to 2 in 87fa759, [`grpc_aio` job](https://app.circleci.com/pipelines/github/DataDog/dd-trace-py/15178/workflows/79ab5b56-e520-4028-af89-d28824f26bf0/jobs/1051739) worked fine. NOTE THAT it only contains test results for 3.7 and 3.9 because tests are splitted across workers and not all of them are shown in UI.

Is this a known issue? If not, I'll create an issue.

Reply:
I see 3.7, 3.8, 3.9, and 3.10 in the CircleCI UI.

You have to scroll through the results from both workers to catch all the pytest runs, we don't have an executive summary at the end of a `riot` run.

Reply:
![Screen Shot 2022-03-09 at 6 52 06](https://user-images.githubusercontent.com/3449164/157331607-098d1024-7e39-4ac0-b381-e19e793edc70.png)
Oh, I didn't know that we could see the results of workers.

Anyway, the reason why CI failed was there were some workers to which valid Python version was not allocated.